Source/Charts/Utils/ChartUtils.swift

```diff
@@ -16,9 +16,26 @@ import CoreGraphics
     import UIKit
 #endif
 
+extension FloatingPoint {
+    var deg2rad: Self {
```

Comment:
Generally it's fine. But do you think we should follow the Camel-Case naming? like deg2Rad

Reply:
Up to you. I tried `deg2Rad` and `deg2RAD` and they both look silly to me. I also considered `inRadians` and `inDegrees`, but that leaves out grads

Reply:
OK. How about we keep old `FDEG2RAD` to keep consitent?

Reply:
I'll drop the F since it's implied as it's an extension on `FloatingPoint`